Utilities/StorageFactory/src/StorageAccount.cc

```diff
@@ -78,35 +140,48 @@ const std::string& StorageAccount::nameForToken( StorageClassToken iToken) {
 
 std::string
 StorageAccount::summaryText (bool banner /*=false*/) {
+  aggregateStatistics();
+
   bool first = true;
   std::ostringstream os;
   if (banner)
     os << "stats: class/operation/attempts/successes/amount/time-total/time-min/time-max\n";
   for (auto i = s_nameToToken.begin (); i != s_nameToToken.end(); ++i) {
```

Comment:
For future reference, a 'range based for loop' would probably make this logic cleaner